builtin/providers/cloudstack/resource_cloudstack_affinity_group.go

```diff
@@ -0,0 +1,136 @@
+package cloudstack
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/xanzy/go-cloudstack/cloudstack"
+)
+
+func resourceCloudStackAffinityGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudStackAffinityGroupCreate,
+		Read:   resourceCloudStackAffinityGroupRead,
+		Delete: resourceCloudStackAffinityGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"type": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
```

Comment:
For consistency with the other resources, could you change the order to `name`, `description`, `type, `project`?